accord-core/src/main/java/accord/topology/Topologies.java

```diff
@@ -64,6 +64,11 @@ default boolean isEmpty()
 
     Set<Node.Id> nodes();
 
+    default Set<Node.Id> nonStaleNodes()
+    {
+        throw new UnsupportedOperationException();
```

Comment:
why?  this can't be accepted for any impl so by not defining it makes it clear to know where to add; no?

Reply:
if you want to block in `Multi` then we should be more explicit about that... and the callers of this should enforce we don't have multi (which is allowed for RX transactions)